traffic_portal/app/src/common/modules/dialog/select/lock/dialog.select.lock.tpl.html

```diff
@@ -39,6 +39,17 @@ <h4 class="modal-title">Create CDN Lock</h4>
                 </div>
             </div>
         </div>
+        <div class="row">
+            <div class="col-sm-12 col-md-12">
+                <div class="form-group">
+                    <label class="control-label" for="sharedUserNames">Usernames to share lock with</label>
+                    <input id="sharedUserNames" name="sharedUserNames" type="text" class="form-control" ng-list ng-model="lock.sharedUserNames" maxlength="256">
+                    <div class="helptext">
+                        A comma separated list of usernames that you want your lock to be shared with.
```

Comment:
Comma-separated lists are kind of clunky -- it would be nice to mirror the list functionality of adding Consistent Hash Query Params in the DS form, where you can add/remove to the list by using buttons. We could even get the list of active users from the TO API and pre-populate the list of valid usernames to choose from.